lib/karafka/instrumentation/vendors/datadog/logger_listener.rb

```diff
@@ -12,11 +12,13 @@ class LoggerListener
           include ::Karafka::Core::Configurable
           extend Forwardable
 
-          def_delegators :config, :client
+          def_delegators :config, :client, :service_name
 
           # `Datadog::Tracing` client that we should use to trace stuff
           setting :client
 
+          setting :service_name
```

Comment:
is there a default for this value or will the default be used when nil?

Reply:
default when `nil`:  https://github.com/DataDog/dd-trace-rb/blob/afc4a025c7c8717050901f1419b7b21856aa9a2a/lib/datadog/tracing/tracer.rb#L128

>  is this change backwards compatible? could there be an explicit default that would match the current behaviour?

so it's backwards compatible. The only difference is that now `nil` will be passed explicitly when not set.